GrpcInterface/Python/rips/instance.py

```diff
@@ -121,6 +124,16 @@ def launch(
         if launch_port != -1:
             requested_port = launch_port
 
+        if not resinsight_executable:
+            filename = Path(sys.prefix) / "share" / "rips" / "rips_config.json"
+            print("Looking for config file", filename)
```

Comment:
Maybe remove this print? Config file will not exist for most users, so it will be confusing?

Reply:
Agree, removed the print statement